template/flake.nix

```
@@ -0,0 +1,22 @@
+{
+  description = "virtual environments";
+
+  inputs.devshell.url = "github:numtide/devshell";
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+
+  outputs = { self, flake-utils, devshell, nixpkgs }:
+    flake-utils.lib.eachDefaultSystem (system: {
+      devShell =
+        let inherit (pkgs.devshell) mkShell importTOML;
```

```suggestion
        let
```
since it's used only once, the indirection is not really worth it IMO.